manticore/core/cpu/abstractcpu.py

```diff
@@ -255,8 +255,10 @@ def resolve_argument(arg):
         descriptors = self.get_arguments()
         argument_iter = imap(resolve_argument, descriptors)
 
+        from ...models import isvariadic
```

Comment:
Moving this import to the top with the others introduces a circular dependency error, which would require some restructuring of things and moving around to fix. Perhaps leave a comment, and gh issue, but do later?